tests/cpp_unit_tests/test_pgm_vnf_converter.cpp

```diff
@@ -90,4 +91,22 @@ TEST_CASE("Test convert_input_wrapper") {
     CHECK(json_string == result);
 }
 
+TEST_CASE("Test parse_vnf_file_wrapper minimal example") {
+    std::ifstream f("C:/Users/laury/Downloads/vision_validation.vnf");
```

Comment:
i will move this code to the CLI.

If possible, please try to never commit stuff like this, cause it may break other people's systems